components/provisioner/templates/cluster-bom.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: "hub.k8s.sap.com/v1"
+kind: ClusterBom
+metadata:
+  name: {{ .ShootName }}
+spec:
+  secretRef: {{ .ShootName }}.kubeconfig
+  applicationConfigs:
+  - id: kyma
+    configType: helm
+    readyRequirements:
```

Comment:
OIDC config could go here

Reply:
Yes, I am waiting to confirm the configuration.